en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,74 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+To give the minimum constraint on contents, Open Orchestra allows you to store any type
+of value in the contentAttribute.
+Sometimes, there is no easy way to render the value to the backoffice user so we have added
+a stringValue.
+The stringValue is a projection of the value in a string.
+
+Transformers
+------------
+
+Adding a transformer
+~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra transform the value in a string thanks to transformer.
+To add a new one, create a service which implements
+``OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer\ValueTransformerInterface``
+with the tag ``open_orchestra_backoffice.value_transformer.strategy`` :
+
+.. code-block:: yml
+
+    open_orchestra_backoffice.value_transformer.array:
+            class: %open_orchestra_backoffice.value_transformer.array.class%
+            tags:
+                - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Then implements ``ValueTransformerInterface`` in your transformer :
+
+- ``transform`` : transform value to an Html string
+- ``support`` : return true if the transformer must be used
+- ``getName`` : use by manager as index for the transformer
+
+.. code-block:: php
+
+    class ArrayToHtmlStringTransformer implements ValueTransformerInterface
+    {
+        public function transform($data)
+        {
+            //transform data to html
+
+            return $htmlData
+        }
+
+        public function support($fieldType, $value)
+        {
+            return gettype($value) == 'array' && $fieldType == 'choice';
+        }
+
+        public function getName()
+        {
+            return 'array';
+        }
+    }
```

Comment:
what is the use of the code if you don't actually implement the transform method, even in an obvious way ?